### ISSUE TITLE
empty preditions for average multiple scale preds

### ISSUE THREAD

**Issue by PigUnderRoof** [DeepLabCut/DeepLabCut#2212]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Bug description

I think it's the scale issue: when the mouse is too large compared to training data, superanimal_mouse_topview model might fail to extract pose, returing a empy list rather than a dictionary with "coordinates" and "confidence" as keys. Then _average_multiple_scale_preds function fails to query information.

I add the three lines before [line 90](https://github.com/DeepLabCut/DeepLabCut/blob/ece1b9da60095370d36b9093a1b3495e09de207d/deeplabcut/modelzoo/api/superanimal_inference.py#L90) to skip such situations:

```python
if not isinstance(pred,dict):
    xyp[scale_id] = np.nan
    continue
```

Besides, would it be a good idea to give the statistics of mouse size in training data, so that we can roughly estimate what values might be suitable for scale_lsit to \_\_init\_\_ SpatiotemporalAdaptation instance.

### Operating System

Ubuntu

### DeepLabCut version

2.3.3

### DeepLabCut mode

single animal

### Device type

gpu

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/DeepLabCut/DeepLabCut/blob/master/CODE_OF_CONDUCT.md)

**Comment by yeshaokai** [comment]
@PigUnderRoof Great idea for adding statistics. Will follow up with you and update it in our document.

**Comment by PigUnderRoof** [comment]
@yeshaokai  thx!

**Comment by MMathisLab** [comment]
@yeshaokai @jeylau  updates on this ?

**Comment by jeylau** [comment]
@MMathisLab, this will be fixed with #2198.
